src/modules/doubt/dto/update-doubt.dto.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  IsBoolean,
+  IsEnum,
+  IsNotEmpty,
+  IsOptional,
+  IsString,
+} from 'class-validator';
+import { TagType } from '../doubt-tag.enum';
+
+export class UpdateDoubtDto {
+  @IsNotEmpty()
+  @IsEnum(TagType)
```

Comment:
Also Add is Array and check if it is working correctly

Reply:
Ok